server/datastore/datastore.go

```diff
@@ -8,8 +8,10 @@ import "time"
 // Any Struct used as the app database must implement these functions.
 type DB interface {
 	Init(config DatabaseConfig) error
-	AllMaintenanceRequests() ([]MaintenanceRequest, error) //Calls to the Requests method within maintenance_requests.go
-	CreateMaintenanceRequest(request MaintenanceRequest) (MaintenanceRequest, error)
+	AllMaintenanceRequests() ([]MaintenanceRequest, error)
+	CreateMaintenanceRequest(request MaintenanceRequest) (*MaintenanceRequest, error)
+
```

Comment:
do you run `go fmt` on your files? Or does your editor? It would've removed extra newlines like this

Reply:
My editor normally does that, but I will double check